pandas/_libs/lib.pyx

```diff
@@ -1624,6 +1624,10 @@ cdef class StringValidator(Validator):
     cdef inline bint is_array_typed(self) except -1:
         return issubclass(self.dtype.type, np.str_)
 
+    cdef bint is_valid_null(self, object value) except -1:
```

Comment:
this should be in missing.pyx with the rest of these. I would also give this a more meaningful name

Reply:
This is overridding the method on the parent Validator class.